src/commands/new.ts

```diff
@@ -180,7 +180,7 @@ module.exports = {
 
     const yname = boolFlag(options.y) || boolFlag(options.yes)
     const noTimeout = options.noTimeout ?? false
-    const useDefault = (option: unknown) => yname && option === undefined
```

Comment:
The new linter config was complaining about "use" being a react-only keyword so i changed it to "get"